samples/gen-js/test/test.js

```diff
@@ -0,0 +1,35 @@
+const assert = require("assert");
```

Comment:
yeah, I think we should eventually write some full E2E tests, but I'm not sure the best way to structure them...